Update the XML files in the ja folder without changing the name attribute values of the XML files in the en folder.

YuzuruJewell opened the pull request:
Update the XML files in the ja folder without changing the name attribute values of the XML files in the en folder.

The files below could not be replaced with Japanese without changing the duplicate name attribute values using my tools.
In order to apply the Japanese translation without changing the XML files in the en folder, I temporarily changed the name attribute value of the xml files, translated the corresponding English to Japanese, and then changed it back to the original name attribute value.
I would like someone to fix the duplicate name attribute values on these files.

floater_about_land.xml
floater_beacons.xml
floater_model_preview.xml
floater_pathfinding_linksets.xml
floater_preferences_graphics_advanced.xml
floater_scene_load_stats.xml
floater_snapshot.xml
floater_test_layout_stacks.xml
floater_test_line_editor.xml
floater_test_text_editor.xml
floater_test_textbox.xml
floater_test_widgets.xml
floater_translation_settings.xml
floater_world_map.xml
fonts.xml
menu_navbar.xml
menu_settings_gear.xml
menu_topinfobar.xml
mime_types.xml
panel_group_creation_sidetray.xml
panel_notification_list_item.xml
panel_performance_preferences.xml
panel_preferences_advanced.xml
panel_preferences_sound.xml
panel_profile_pick.xml
panel_volume_pulldown.xml
sidepanel_item_info.xml

Ansariel commented on indra/newview/skins/default/xui/ja/floater_aaa.xml:
Adding tabs to additional lines will show escape characters in the UI.

Ansariel commented on indra/newview/skins/default/xui/ja/floater_auction.xml:
The nodes were already valid - no need for this change.

Ansariel commented on indra/newview/skins/default/xui/ja/floater_beacons.xml:
```diff
@@ -1,27 +1,38 @@
-		<check_box label="音源" name="sounds"/>
-		<check_box label="パーティクル源" name="particles"/>
-		<check_box label="メディアソース" name="moapbeacon"/>
+		<check_box label="物理的" name="physical">
```
Unnecessary changes again.

Ansariel commented on indra/newview/skins/default/xui/ja/floater_beacons.xml:
```diff
@@ -1,27 +1,38 @@
+		<text tool_tip="ビーコンの幅" name="beacon_width_label">
 			幅:
 		</text>
+		<slider initial_value="1" name="beacon_width"/>
```
Overrides value outside text attributes.

Ansariel commented on indra/newview/skins/default/xui/ja/floater_buy_currency_html.xml:
```diff
@@ -1,2 +1,6 @@
-<floater name="floater_buy_currency_html" title="通貨の購入"/>
\ No newline at end of file
+<floater name="floater_buy_currency_html" title="通貨の購入">
+	<floater.string name="buy_currency_url">
```
Not language specific, therefore shouldn't be overridden.

marchcat-pe commented:
Thank you, @YuzuruJewell!
The patch is huge, so I'll review it when able.

YuzuruJewell commented:
I removed the working attributes left in floater_test_text_editor.xml and removed the attributes with empty attribute values.
I also fixed incorrect indentation of node text.
There may still be some translation results that are the same as English. I will fix them in future update.

YuzuruJewell commented:
Thank you, Ansariel and marchcat-pe.
I made the following updates.
- The representations of nodes without text were updated.
- The XML files that had no changes were deleted.
- The attributes of the Japanese XML files with the same attribute value as English were removed.
- The text indentation of "text" and "text_editor" nodes were changed from tabs to spaces.
Please check them.

* I committed the same file twice. I tried deleting the later commit, but I couldn't do it.
  Please delte later commit.

marchcat commented:
> * I committed the same file twice. I tried deleting the later commit, but I couldn't do it.
>   Please delte later commit([9c2fdc3](https://github.com/secondlife/viewer/commit/9c2fdc3003129052940f59d829c2e98e1a09a7f9)). "_ja" folder must be deleted.

Done! Closing this PR as it's now part of #2232.

Thank you again for your contribution!
